### DISCUSSION TITLE
Azure Webapp - Publish from VS2019 Works, Publishing from Devops does not

### DISCUSSION THREAD

**Discussion by mapelli0** [cefsharp/CefSharp#3794] [General]
Hello, sorry to ask on here, I am not sure if this is the right place.
I am trying to deploy a project that uses CEFSharp on azure webapp
If I publish using VS2019 it works flawlessly, if I use the exact same code, build and publish from Pipeline and Release from devops, it breaks the initialization with "An attempt was made to load a program with an incorrect format."

Have anyone had the same problem?
I tried checking if all the files are there, and apparently the publish from devops doesn't copy all the correct files, but even if I manually do so or use copy always, the process still can't get initialized.

**Comment by amaitland** [comment]
Are you trying to run your application on azure? It's not something I've tried personally. 

There is a similar discussion at https://stackoverflow.com/questions/66705939/cefsharp-offscreen-netcore-with-azure-function

Does azure support running Visual C++? Does azure support running chromium? You might need to reach out to their support to confirm the requirements are met.

**Comment by mapelli0** [comment]
Yes, it works, when I publish direct from VS2019
When I do it from devops (pipeline + release) it does not.

**Reply by amaitland** [reply]
No idea then. Try a devops support channel, ask for how to debug.

Post a link back here for reference.

**Comment by mapelli0** [comment]
I added this question over stackoverflow, if anyone want to contribute
https://stackoverflow.com/questions/69176865/cefsharp-deploy-using-azure-devops
